charts/garden-project/charts/project-rbac/templates/clusterrole-project-admin.yaml

```diff
@@ -2,9 +2,9 @@
 apiVersion: {{ include "rbacversion" . }}
 kind: ClusterRole
 metadata:
-  name: garden.sapcloud.io:system:project:{{ .Values.project.name }}
+  name: gardener.cloud:system:project:{{ .Values.project.name }}
```

Comment:
Changing the name here will result in orphaned resource?
Basically I have tried with helm(I know we are not using it for this chart particularly)
So, with `helm install` the old resource leaks, with `helm upgrade --install --force` the old one is deleted.


PS. At the time I wrote this I have not seen the whole PR.

Reply:
Yes, good point, I have implemented clean up logic in the project controller.